code/game/objects/items/storage/backpack.dm

```diff
@@ -292,15 +289,40 @@
 			select_gamemode_skin(type)
 		..()
 
+/obj/item/storage/backpack/marine/standard
+	name = "\improper lightweight IMP backpack"
+	desc = "The standard-issue pack of the USCM forces. Designed to slug gear into the battlefield."
+	bypass_w_limit = list("/obj/item/device/m56d_gun",
```

Comment:
Standard backpacks don't need this.

Reply:
All backpacks should have this capability.

Reply:
Not really.

Reply:
The engineering backpack provides an engineer-relevant function. That is good. The rest don't.